ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -97,10 +125,15 @@
                        :internal-error
                        [(str "There was a problem saving a bulk granule update request."
                              "Please try again, if the problem persists please contact cmr-support@earthdata.nasa.gov.")]))))]
-    ;; Queue the granules bulk update event
-    (ingest-events/publish-gran-bulk-update-event
+
+    ;; Queue the granules bulk update events
+    (publish-instructions-partitioned
      context
-     (ingest-events/granules-bulk-event provider-id task-id user-id instructions))
+     provider-id
+     user-id
+     task-id
+     instructions
+     (granule-bulk-update-chunk-size))
```

Comment:
The previous limiter seemed to be the message size. Partition the message into smaller bites to process

Reply:
I don't think parsing out the original request and queue the granule events in chunks is necessary.

Reply:
I like the way you queue the granule bulk update events.